package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aperture.js",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Don't bump. This will be done by one of the maintainers when they're ready to do a release.